### PR TITLE
conda-suggest-conda-forge

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
This is a simple data package that is ready for review @conda-forge/staged-recipes

File: recipes/conda-suggest-conda-forge/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+build:
+  number: 0
+
+test:
+  commands:
+    - test -f $PREFIX/share/conda-suggest/conda-forge.noarch.map  # [unix]
+    - test -f $PREFIX/share/conda-suggest/conda-forge.{{ build_platform }}.map  # [unix]
+    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.noarch.map exit 1  # [win]
+    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.{{ build_platform }}.map exit 1  # [win]
```
Comment (isuruf) [inline recipes/conda-suggest-conda-forge/meta.yaml:20]:
```suggestion
    - if not exist %PREFIX%\\share\\conda-suggest\\conda-forge.{{ target_platform }}.map exit 1  # [win]
```

Comment (scopatz) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: I can fix this up in the feedstock, but target_platform doesn't seem to be listed in the conda-build docs as the way to get the subdir name.  Where is this coming from? conda-build-config?

**Comment by scopatz** [comment]
Thanks @beckermr!

File: recipes/conda-suggest-conda-forge/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: It's from `conda-build`. `build_platform` is where this package is built in. `target_platform` is where the package ends up in.

Comment (scopatz) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: It seems to be undocumented, so probably not safe to use

Comment (isuruf) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: `build_platform` is wrong here though.

Comment (isuruf) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: Can you open an issue in `conda-build` to document `target_platform`?

Comment (scopatz) [reply recipes/conda-suggest-conda-forge/meta.yaml:20]: I agree, but as long as build and target are the same, it should be OK. I am a little overwhelmed right now trying to get through a rather large checklist.

**Pull request merged**